data/update-scriptworker-revisions.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
```

Comment:
I feel like this should be under `modules/signing_worker` somewhere. Possible just alongside the requirements.txt files.

Reply:
Ah sorry, I didn't realize that's where the revisions were. Why is that, do other modules need scriptworker?

Reply:
This is before my time, but for what I understand `/data/*` has all values to feed into the puppet scripts. So, pretty much every script will pull data from these files.

The update script only touches `data/common.yaml`, so I decided it should live right beside it. Same logic for the `modules/signing_worker/files/update-requirements.sh` script, as it's only modifying files in the same directory. Not sure if it would make sense for a script in `modules/signing_worker/` to modify `data/common.yaml`.

Reply:
Merging as is. If we decide to move this around, it should be in a separate PR.